internal/databases/postgres/daemon_handler.go

```diff
@@ -166,3 +168,25 @@ func Listen(c net.Conn, uploader *WalUploader) {
 		}
 	}
 }
+
+const SdNotifyWatchdog = "WATCHDOG=1"
+
+func SdNotify(state string) error {
+	socketName, ok := os.LookupEnv("NOTIFY_SOCKET")
```

Comment:
Use the viper config setting instead. You can check how other settings are implemented for reference.

Reply:
Also, it is better to skip the notification if the config setting is unset.